ducktape/tests/test.py

```diff
@@ -250,6 +250,15 @@ def configure_logger(self):
         info_fh.setFormatter(formatter)
         debug_fh.setFormatter(formatter)
 
+        if self.session_context.debug:
+            # Colorize output if debugging is enabled
+            # FIXME: Might want to check for terminal capability too.
+            logging.addLevelName(logging.CRITICAL, "\033[41;33mCRIT ")
```

Comment:
Output alignment and screen estate frugality, otherwise the timestamp position will slip around depending on logging level which is annoying.